nailgun/entities.py

```diff
@@ -1348,7 +1348,8 @@ def __init__(self, server_config=None, **kwargs):
         super(GPGKey, self).__init__(server_config, **kwargs)
 
 
-class HostClasses(Entity):
+class HostClasses(
+        Entity, EntityCreateMixin, EntityDeleteMixin, EntityReadMixin):
     """A representation of a Host Class entity."""
 
     def __init__(self, server_config=None, **kwargs):
```

Comment:
Below, I see a reference to `self.host`, but there's no guarantee that `self.host` will exist. How about calling `_check_for_value` here?

```python
def __init__(self, server_config=None, **kwargs):
    _check_for_value('host', kwargs)
    self._fields = {…}
    super(HostClasses, self).__init__(server_config, **kwargs)
    self._meta = {
        'server_modes': ('sat'),
        # pylint:disable=no-member
        'api_path': '{0}/puppetclass_ids'.format(self.host.path('self'))
    }
```